go/autoscaler/k8s/cluster.go

```diff
@@ -12,20 +26,20 @@ import (
 	batchv1 "k8s.io/client-go/pkg/apis/batch/v1"
 )
 
-// K8sCluster resprensents a Kubernetes cluster.
-type K8sCluster struct {
+// Cluster resprensents a Kubernetes cluster.
+type Cluster struct {
 	clientset *kubernetes.Clientset
 }
 
-// NewK8sCluster create a new instance of K8sCluster.
-func NewK8sCluster(clientset *kubernetes.Clientset) *K8sCluster {
-	return &K8sCluster{
+// NewCluster create a new instance of K8sCluster.
+func NewCluster(clientset *kubernetes.Clientset) *Cluster {
```

Comment:
Putting `K8sCluster` into `k8s` package, so it can be renamed to `Cluster`. From outside of the package, it will be referred as `k8s.Cluster`.